data-prepper-core/src/main/java/org/opensearch/dataprepper/DataPrepper.java

```diff
@@ -71,6 +75,28 @@ public DataPrepper(
      */
     public boolean execute() {
         peerForwarderServer.start();
+        Set<String> pipelineNames = transformationPipelines.keySet();
```

Comment:
This could use a better name to improve readability. Perhaps `waitingPipelineNames`?